lib/driverProviders/sauce.js

```diff
@@ -5,7 +5,7 @@
  */
 
 var util = require('util'),
-    log = require('../logger.js'),
+    log = require('../logger.js').Logger,
```

Comment:
In logger.js, looks like it would require changing the module to a class to use `export default`. Trying that out, it looks like `default` would make calls like `log.set(` change to `log.default.set(` for javascript files like this one.